Bookish.DataAccess/BookishService.cs

```diff
@@ -87,5 +90,39 @@ ORDER BY
 
             return dbConnection.Query<BookLoan>(query, new { UserID = userId });
         }
+
+        public void AddBook(string title, string author, string isbn, int totalCopies)
+        {
+            var lastId = LastCopiesId();
+
+            var valuesList = new List<string>();
+            for (var i = 1; i <= totalCopies; i++)
+            {
+                valuesList.Add($"({lastId + i}, @ISBN)");
+            }
+
+            var query =
+                @"INSERT INTO Books (ISBN, Title, Author)
+                VALUES (@ISBN, @Title, @Author); " +
+                @"INSERT INTO BookCopies (CopyId, ISBN) VALUES " +
+                string.Join(", ", valuesList);
+
+            dbConnection.Execute(query, new { Title = title, Author = author, ISBN = isbn });
+        }
+
+        public bool GetIsbnExists(string isbn)
+        {
+            var query = "SELECT ISBN FROM Books WHERE ISBN = @ISBN";
+
+            return dbConnection.Query<string>(query, new {ISBN = isbn})
```

Comment:
Minor but should have whitespace after opening curly bracket and before closing curly bracket